maps/torch/items/summarydocuments.dm

```diff
@@ -1,130 +1,114 @@
+	desc = "A \"for official use only\" memorandum detailing an officer's options with regards to administrative action as referenced in the disposition of offenses. Options include: counseling, corrective training, administrative reprimands, bar to reenlistment, suspension of favorable personnel actions (flagging), MOS reclassification, revocation of security clearance, relief from duties, adverse NCOER/OER, removal from special status, rehabilitative transfer, administrative reduction, and administrative separations."
 
 /obj/item/documents/generic/sol/njprules
-	name = "nonjudicial punishments"
-	desc = "A \"for official use only\" memorandum detailing options with regards to nonjudicially punishing uniformed servicemembers. Such punishments include admonishment/reprimand, forfeiture of pay, restriction, extra duty, reduction in rank, and correctional custody if the installation has an approved facility. Amount of punishment depends on rank of accused and rank of commander."
-
-/obj/item/documents/generic/sol/enlistednjp
-	name = "nonjudicial punishments for enlisted personnel"
-	desc = "A \"for official use only\" memorandum detailing options with regards to nonjudicially punishing enlisted uniformed servicemembers. For a summarized NJP administered aside from on-the-spot correction, there can be no forfeiture of pay or reduction of rank. Restriction and extra duty is capped at fourteen days. For a company-level NJP, forfeiture of one weeks� pay is authorized. Reduction is capped at one paygrade reduction (E1-4). Restriction and extra duty are capped at two weeks. Field Grade NJPs are reserved for egregious violations. Forfeiture up to half of one-month�s pay for two months is authorized. Reduction in rank for E1-4 can be total. For E5-E6, only one rank reduction is authorized. Restriction of up to 60 days and extra duty of up to 45 days is authorized."
-
-/obj/item/documents/generic/sol/officernjp
-	name = "nonjudicial punishments for commissioned personnel"
-	desc = "A \"for official use only\" memorandum detailing options with regards to nonjudicially punishing commissioned uniformed servicemembers. There is no summarized NJP process. For a company-level NJP, a written reprimand/admonition is required. Restriction is capped at thirty days. Field Grade NJPs are the same. General Officer NJPs require a written letter of reprimand or admonition. Punishment includes up to 60 days restriction or 30 days arrest in quarters. Forfeiture of half of one-month�s pay for two months is authorized."
```

Comment:
Tried toying a lot with this to make it as concise as possible but it just didn't work. Plus a lot of the information here directly contradicts the wiki, which should be seen as the primary source of SCUJ.

Reply:
Eckles ruled directly (iirc) that this was not going to get in anyway, unless it was as per the wiki.
In which case, it would just be a wiki doc pasted directly, which is dumb.

Agreed.